src/rsa.rs

```diff
@@ -86,9 +92,29 @@ impl signature_impl::VerificationAlgorithmImpl for RSA_PKCS1 {
     }
 }
 
+macro_rules! rsa_pkcs1_padding {
+    ( $PADDING_ALGORITHM:ident, $digest_alg_name:expr,
+      $digest_alg:expr, $digestinfo_prefix:expr ) => {
+
+        #[doc="PKCS#1 1.5 padding with the "]
+        #[doc=$digest_alg_name]
+        #[doc=" digest algorithm."]
```

Comment:
Never mind the second part. I see that the `RSA_PKCS1_3072_8192_SHA384_VERIFY` would screw this up.